components/toolbar/style.scss

```diff
@@ -88,8 +88,14 @@ div.components-toolbar {
 	}
 
 	// active & toggled style
-	&:not(:disabled).is-active > svg {
-		@include formatting-button-style__active;
+	&:not(:disabled).is-active{
+		& > svg {
+			@include formatting-button-style__active;
+		}
+
+		&[data-subscript]:after {
+			color: $white;
```

Comment:
Curious: Should this just `@include formatting-button-style__active;` as well? Who's to say that this is `white`? It's only white because we assume its should align with the mixin, but that's a fragile dependency.

Reply:
Yes, I agree with you, to colors should not repeated as it increases maintainability problems and makes causing regressions easier. Just reusing the existing mixin was not possible because it broke the styles I opted to create a specific mixin for the color so no repetition exists.